index.html

```diff
@@ -6,8 +6,36 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <title>Portfolio</title>
+    <link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+
+
 </head>
-<body>
-    
+<body class="container">
+    <div class="toolbar">
+        <h2 class="logo">Wahid Shadab</h2>
+        <div class="menu">
+            <div class="menu_item"></div>
+            <div class="menu_item"></div>
+            <div class="menu_item"></div>
+        </div>
+    </div>
```

Comment:
- The container of the section is a header tag.
- The logo (if the design has one) is on the left, and has a link (a tag).
- The hamburger button is on the right, and has a nav tag.